packages/cli/src/__tests__/commands/DebugInfo.test.ts

```diff
@@ -149,14 +149,106 @@ describe('debug', () => {
       false
 
       -- CI detected? --
-      No
+      false
+
+    `)
+  })
+
+  it('should succeed when env vars are set to empty', async () => {
+    ctx.fixture('example-project/prisma')
+
+    // Make sure all env vars are set to ''
+    const envVarsSetToEmptyString = Object.fromEntries(Object.keys(envVars).map((key) => [key, '']))
+    Object.assign(process.env, envVarsSetToEmptyString)
+    // To make sure the terminal is always detected
+    // as non interactive, locally and in CI
+    process.env.TERM = 'dumb'
+
+    const result = await DebugInfo.new().parse([])
+
+    expect(cleanSnapshot(result as string)).toMatchInlineSnapshot(`
+      -- Prisma schema --
+      Path:  REDACTED_PATH
+
+      -- Local cache directory for engines files --
+      Path:  REDACTED_PATH
+
+      -- Environment variables --
+      When not set, the line is dimmed and no value is displayed.
```

Comment:
Does this look intuitive with the backticks when seen in an actual terminal?

Reply:
Well, we need to print a few different things, keep in mind that env vars are always strings.
- `undefined` value, instead of printing undefined we leave this empty, because technically you could also pass the string `undefined` as a value.
- empty string ``, the env var is set to an empty string value
- some string `1`, the env var is set to a string value
    - some string with only blank spaces `  `   
	- some string `DEBUG="*" command` (random example showing that quotes are theoretically possible)  

So adding backticks helps here to distinguish what's actually the value.

Reply:
So in the test all the env vars are set to empty string?

Reply:
I added tests for the different possibilities here, not only empty.